routes/voyager.php

```diff
@@ -129,6 +129,16 @@
             Route::post('/', ['uses' => $namespacePrefix.'VoyagerCompassController@index',  'as' => 'post']);
         });
 
+        //Profile
+        Route::group([
+            'as'     => 'users.',
+            'prefix' => 'profile',
+        ], function () use ($namespacePrefix) {
+            Route::get('/{id}', ['uses' => $namespacePrefix.'VoyagerController@showProfile', 'as' => 'showProfile']);
```

Comment:
The inclusion of `id` seems unnecessary.  `profile` implies `my user account`.  Additionally, you only allow access to the current user in the VoyagerController methods you've added